Package.swift

```diff
@@ -13,24 +13,38 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.4.0"),
-        .package(url: "https://github.com/maiqingqiang/Jinja", from: "1.0.6")
+        //        .package(url: "https://github.com/maiqingqiang/Jinja", from: "1.0.6"),
```

Comment:
This URL needs to be adjusted, change maiqingqiang to johnmai-dev.

https://github.com/johnmai-dev/Jinja

@DePasqualeOrg